### DISCUSSION TITLE
PM command

### DISCUSSION THREAD

**Discussion by ghost** [FakeFishGames/Barotrauma#9030] [Ideas and suggestions]
I think a command for server moderators that lets them directly send messages to individual players would be quite useful. I find myself having to send a message to the entire server when one person starts toeing the rules. Being called out for everyone to see is also quite embarassing.

Letting the existing message feature for spectators be used by everyone could also work. Maybe make it a server option, since some people would want it disabled.

**Comment by James-E-A** [comment]
Alternative suggestion: a "Whisper" chat command for local messages that are stopped by waterproof walls and die out rapidly past 400 units

`;w Hey, you need to stop growing mudraptors in the ballast or I will dox you`

**Reply by ghost** [reply]
I think both would be great, I like the idea of more ways to talk without others being able to hear it.

**Comment by cshingle** [comment]
This is why local chat is a thing. As a captain I ask people to come to navigation and talk to them on local chat. On my new sub I will ask them to my office.

**Reply by ghost** [reply]
That's good and all, but what if I'm spectating my server?
